### PR TITLE
CI for push & PR using gradle and Github actions

### PR DESCRIPTION
**Pull request by svillar**
This is a temptative implementation of CI for push and PRs using gradle and github actions

### CODE REVIEW

**Comment by pmatos** [comment]
LGTM as long as the secret `THIRD_PARTY_PAT` is defined in the project settings with the correct checkout token.

**Comment by pmatos** [comment]
Also, the CI seems to be running and green so another evidence that the secrets stuff is working.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    steps:
+      - name: Checkout Wolvic
+        uses: actions/checkout@v2
+        with:
+          submodules: 'true'
+
+      - name: Checkout third parties
+        uses: actions/checkout@v2
+        with:
+          repository: Igalia/wolvic-third-parties
```
Comment (MortimerGoro) [inline .github/workflows/ci.yml:26]: We use this file to associate a Wolvic version with a third_party commit: https://github.com/Igalia/wolvic/blob/main/third_party_hash

Would it be possible to take the commit specified in that file?

Comment (MortimerGoro) [reply .github/workflows/ci.yml:26]: This can be done as a follow-up, main branch is fine for now

**Review by MortimerGoro — APPROVED** [review]
LGTM, can you delete the unused files from taskcluster?

**Comment by clopez** [comment]
LGTM

File: .github/workflows/ci.yml (hunk 1 above)
Comment (svillar) [reply .github/workflows/ci.yml:26]: OK, I guess it should be doable.

**Pull request merged**